docker-bake.hcl

```diff
@@ -1,21 +1,29 @@
-group "default" {
-  targets = ["python"]
+variable "BUILD_VERSION" {
+  default = "dev"
+}
+
+variable "PYTHON_VERSION" {
+  default = "3.13.5"
 }
 
-variable "PYTHON_TAG" {
-  default = "3.13.5-slim-bookworm"
+variable "PYTHON_BASE_VARIANT" {
+  default = "slim-bookworm"
 }
 
-variable "PYTHON_SHA" {
+variable "PYTHON_BASE_SHA" {
   default = "f2fdaec50160418e0c2867ba3e254755edd067171725886d5d303fd7057bbf81"
 }
 
+group "default" {
+  targets = ["python"]
+}
+
 target "python" {
   context = "docker/python"
   args = {
-    PYTHON_TAG_SHA = "${PYTHON_TAG}@sha256:${PYTHON_SHA}"
+    PYTHON_TAG_SHA = "${PYTHON_VERSION}-${PYTHON_BASE_VARIANT}@sha256:${PYTHON_BASE_SHA}"
   }
   tags = [
-    "ghcr.io/powerhome/cdk8s-cmp-python:${PYTHON_TAG}"
+    "ghcr.io/powerhome/cdk8s-cmp-python:${BUILD_VERSION}-${PYTHON_VERSION}"
```

Comment:
Why not simplify?

```suggestion
    "ghcr.io/powerhome/cdk8s-cmp:${BUILD_VERSION}"
```

Reply:
Are you suggesting something like `cdk9s-cmp:v1.0.0-python`, or just `cdk9s-cmp:v1.0.0`. We will want to build the TypeScript, Go, and Java images in this repo as well. As for python versions 3.13.5 on the tag, originally I envisioned supporting different Python versions. But I'd be okay just supporting 3.13 and up through regular versioning if you think thats better.

Reply:
I can see heavy TypeScript usage at the very least.